docs/source/changelog.rst

```diff
@@ -68,6 +68,7 @@ Mobjects, Scenes, and Animations
 #. :code:`Scene` now renders when :code:`Scene.render()` is called rather than upon instantiation.
 #. :code:`ValueTracker` now supports increment using the `+=` operator (in addition to the already existing `increment_value` method)
 #. Add :class:`PangoText` for rendering texts using Pango.
+#. The ``alignment`` attribute to Tex and MathTex has been removed in favour of ``tex_environment``.
```

Comment:
We should introduce a new section "next release" or so above the v0.1 section in the `changelog.rst`, and this line should be moved there.

Reply:
I added a new section for 0.2.0 ("changes since v0.1.0")